README.md

```diff
@@ -1426,6 +1427,36 @@ $ export AIRPLANE=true
 $ npm test
 ```
 
+## Contributors
+
+This project exists thanks to all the people who contribute. See guidelines above. 
+<a href="graphs/contributors"><img src="https://opencollective.com/nock/contributors.svg?width=890&button=false" /></a>
```

Comment:
@RichardLitt I’d say we leave that section?

Reply:
Looks fine to me.